zhaquirks/tuya/builder/__init__.py

```diff
@@ -99,35 +107,54 @@ def tuya_battery(
         self.adds(power_cfg)
         return self
 
-    def tuya_metering(
-        self,
-        dp_id: int,
-        metering_cfg: TuyaLocalCluster = TuyaValveWaterConsumed,
-    ) -> QuirkBuilder:
-        """Add a Tuya Metering Configuration."""
-        self.tuya_dp(
-            dp_id,
-            metering_cfg.ep_attribute,
-            "current_summ_delivered",
+    def tuya_contact(self, dp_id: int):
+        """Add a Tuya IAS contact sensor."""
+        self.tuya_ias(
+            dp_id=dp_id,
+            ias_cfg=TuyaIASContact,
+            converter=lambda x: IasZone.ZoneStatus.Alarm_1 if x == 1 else 0,
```

Comment:
I assume most contact sensors will only push `0` and `1` as values, but in case some sensor uses `2` (e.g. if two alarms are triggered), we might wanna catch that too and still trigger the alarm.

So, we could have any non-0 value should activate the alarm.
It would be similar to the binary sensor parsing for the ZCL sensors defined in ZHA then, as they just use `bool(value)`.

```suggestion
            converter=lambda x: IasZone.ZoneStatus.Alarm_1 if x != 0 else 0,
```